### ISSUE TITLE
Attack Gym

### ISSUE THREAD

**Issue by ghost** [FeroxRev/Pokemon-Go-Rocket-API#75]
Can someone give me an example of the battle id and the battleactions in the attack gym
thank you

**Comment by vmcall** [comment]
I dont think the current proto supports StartGymBattle (https://github.com/FeroxRev/Pokemon-Go-Rocket-API/issues/36)

Edit: MIGHT work since deployfortpokemon is working thanks to the proto update

**Comment by Bloontede** [comment]
Just tried it and StartGymBattle response is empty every time. Any hope for a fix? :)

**Comment by chrisoro** [comment]
Yup, same here. StartGymBattle is empty, so cannot start an attack :(

**Comment by Bloontede** [comment]
Compiling the latest protos fixed the method (defendingpokemonid changed to fixed64).

**Comment by chrisoro** [comment]
Nice, thanks for your input! :)

**Comment by vmcall** [comment]
Would you mind sharing the latest protos compiled?

**Comment by Bloontede** [comment]
Sure, here https://www.wetransfer.com/downloads/a8e0de530e0eb8e1e7e4241ff22922ee20160731223201/1af7c9

**Comment by chrisoro** [comment]
Did it work for you? Mine is still not working with protos 1.5

edit: Nvm, I misscalculated the attackerlist..

Ticket can be closed

**Comment by JDevjs** [comment]
guys plz can you help i want to learn how to use and edit these files so i can help the community but i need like a video or something to help me out start with this API

**Comment by uz19ibyd** [comment]
Just found that issue as i was playing around with the battling Logic. I got the startgymbattle working and am a little stuck now with the creation of the <BattleAction> List, required for the AttackGym function. Could somehow give me a hint where i get the parameters like "durationMs", "energyDelta" from ?
Or is there a function that automatically creates the <BattleAction> List?

Thank you in advance!

**Comment by jamie2016-1** [comment]
Can someone provide the fixed protos build, as mentioned by @Bloontede. His link is no longer working.

Also which Protos is used to make the build. I find many Protos under Git.

NVM I sorted it myself.

**Comment by kelvinnn** [comment]
@jamie2016-1 - do you mind sharing sample code of StartGymBattle?
